pkg/sdk/client_integration_test.go

```diff
@@ -3,7 +3,10 @@ package sdk
 import (
 	"context"
 	"database/sql"
+	"encoding/json"
```

Comment:
let's add an example usage in one of the resources and add one test from the terraform perspective (please keep in mind some nice helper func for tracking metadata creation)

Reply:
Done